### DISCUSSION TITLE
Used to Yaml

### DISCUSSION THREAD

**Discussion by kelvin-chappell** [guardian/cdk#180] [Q&A] [answered]
I haven't got a use-case at the moment but I'm just curious how would you use this repo if you're used to building Cloudformation templates in Yaml? 
Have you any showcases or a 'getting started' section perhaps?
I realise it's early days and you're probably focused on development!

**Comment by jamie-lynch** [comment, marked as answer]
Hi Kelvin 👋 

We've started a couple of docs in this area. There's a [general guide to getting started](https://github.com/guardian/cdk/blob/main/docs/001-general-usage.md) as well as specific docs for [starting a new project](https://github.com/guardian/cdk/blob/main/docs/002-starting-a-new-project.md) and [migrating existing stacks](https://github.com/guardian/cdk/blob/main/docs/003-migrating-existing-stacks.md). This is something that we'd like to expand more so do let us know if there is anything in particular you would find useful. 

Something else you might be interested in is the [CDK CLI](https://github.com/guardian/cdk-cli) which we're working on in parallel. That has commands to generate some of the boilerplate to help you getting started. In particular, the `init` function can be used to generate the `cdk` directory with everything you need to get started. (I've just noticed that the README needs updating to show some of the newer commands - I'll get that done ASAP).

**Reply by kelvin-chappell** [reply]
Thanks for the info @jamie-lynch.  The project looks very promising!  I'll give it a go next time I have to do some cloudforming.

**Comment by akash1810** [comment]
[This video](https://www.youtube.com/watch?v=bTC8XV5aLTo) describes how to gracefully migrate/adopt a YAML file into a CDK stack. We tried this method in AMIgo (https://github.com/guardian/amigo/pull/598) to great effect and we can recommend it as a process. It's fairly long winded however and we want to build tooling to make this process easier.
